src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -9,6 +9,174 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        public static void MatTimesSrc(bool tran, bool add, AlignedArray mat, AlignedArray src, AlignedArray dst, int crun)
```

Comment:
(probably for future clean up, like other naming suggestions)  `tran` doesn't really describe what this is.  It should probably be `transpose`.

Reply:
Agree and thanks - documented this in https://github.com/briancylui/machinelearning/issues/2.